Stack/js/index.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Create new 'ul' element
+ * @param {object} parent
+ * @returns {Element}
+ */
+function createUl(parent) {
+  let ul = document.createElement('ul');
+  let line = document.createElement('hr');
+
+  parent.appendChild(ul);
+  parent.appendChild(line);
+
+  return ul;
+}
+
+/**
+ * Show stack
+ * @param arr
+ */
+function stack(arr) {
+  const parent = document.getElementById('stack');
+  let ul = createUl(parent);
+  let count = arr.length;
+  let li;
+
+  for (let id = 0; id < count; id += 1) {
```

Comment:
If you need to use count variable only for loop, you can get length of arr in the loop, like:
```for (let id = 0; id < arr.length; id += 1) {```
And you don't need allocate memory for it